src/test/java/core/basesyntax/model/FruitTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTest {
```

Comment:
we don't tests model classes unless they have some additional buisness logic

Reply:
Cleared